FrameworkSystem/Service/UserDB.py

```diff
@@ -0,0 +1,60 @@
+""" A test DB in DIRAC, using MySQL as backend
+"""
+
+from DIRAC.Core.Base.DB import DB
+
+from DIRAC import gLogger, S_OK, S_ERROR
+
+
+class UserDB(DB):
+  """ Database system for users """
+
+  def __init__(self):
+    """
+    Initialize the DB
+    """
+    DB.__init__(self, 'UserDB', 'Framework/UserDB', 10)
```

Comment:
* Prefere `super`
* c'est quoi ce 10 ?

Reply:
- Ok pour `super` c'est change, ca sera au prochain commit, je m'etais base sur l'exemple de la DOC Dirac, je modifierai dans la doc l'exemple aussi.
- Le 10 c'est le nombre max de requetes qui peuvent etre en attente, la doc DIRAC conseille cette valeur

Pour les 2 points, cf http://dirac.readthedocs.io/en/integration/DeveloperGuide/AddingNewComponents/DevelopingDatabases/index.html#develop-the-database